### PR TITLE
Add possibility to restore use default values for product or category

### PR DESCRIPTION
**Pull request by xcom-magento**
Added the possibility to set the entity in the console command. With _--entity=product/category_ the entity can be specified.

### CODE REVIEW

File: Console/Command/RestoreUseDefaultValueCommand.php [hunk 1]
```diff
@@ -35,7 +36,8 @@ protected function configure()
             ->setDescription("
                 Restore product's 'Use Default Value' if the non-global value is the same as the global value
             ")
-            ->addOption('dry-run');
+            ->addOption('dry-run')
+            ->addOption('entity', null, InputOption::VALUE_REQUIRED, "Set entity to cleanup (product or category)");
```
Comment (sprankhub) [inline Console/Command/RestoreUseDefaultValueCommand.php]:
```suggestion
            ->addOption(
                'entity',
                null,
                InputOption::VALUE_OPTIONAL,
                'Set entity to cleanup (product or category)',
                'product'
            );
```

I think we should keep the current behaviour as the default and only add this as an additional option.

File: Console/Command/RestoreUseDefaultValueCommand.php [hunk 2]
```diff
@@ -72,7 +80,7 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         foreach ($tables as $table) {
             // Select all non-global values
-            $fullTableName = $db->getTableName('catalog_product_entity_' . $table);
+            $fullTableName = $db->getTableName('catalog_'. $entity. '_entity_' . $table);
```
Comment (sprankhub) [inline Console/Command/RestoreUseDefaultValueCommand.php]:
```suggestion
            $fullTableName = $db->getTableName('catalog_' . $entity . '_entity_' . $table);
```

File: composer.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "magento/magento-composer-installer": "*"
   },
   "type": "magento2-module",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (sprankhub) [inline composer.json]: Could you please completely remove the version (see https://blog.packagist.com/tagged-a-new-release-for-composer-and-it-wont-show-up-on-packagist/)?

**Review by sprankhub — CHANGES_REQUESTED** [review]
Thanks for your contribution! I hope my comments make sense to you :)

File: composer.json (hunk 3 above)
Comment (xcom-magento) [reply composer.json]: We've removed the version from the composer.json.

**Review by sprankhub — APPROVED** [review]

**Pull request merged**